dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ExclusionFiles.java

```diff
@@ -82,7 +82,6 @@
  *   <li>A FieldMatcher matches field symbol specified by className and name attribute.
  * </ul>
  */
-@SuppressWarnings("deprecation")
```

Comment:
I see it is for XMLReaderFactory.

<img width="824" alt="Screen Shot 2021-01-12 at 10 12 03" src="https://user-images.githubusercontent.com/28604/104332884-a4c7a980-54be-11eb-8e6a-4f0bae69b2dc.png">

How about moving this to the `createXmlReader()` method (a smaller scope than the entire class)?